codec-itch/README.md

```diff
@@ -0,0 +1,22 @@
+# ITCH codec
+
+This codec can be used for decoding and encoding message in ITCH protocol
+
+## Configuration
+
+### Codec factory
+
+To use the following codec you need to specify the following codec factory:
```

Comment:
To use the **subsequent** codec **,** you need to specify the following codec factory: